.github/CODEOWNERS

```diff
@@ -24,6 +24,7 @@
 /src/lib/util/Notifications/ @quantumlytangled
 /src/lib/types/settings/raw/RawTwitchStreamSubscriptionSettings.ts @quantumlytangled
 /src/lib/types/definitions/Twitch.d.ts @quantumlytangled
+/docker/ @quantumlytangled
```

Comment:
```suggestion
/docker/influxdb/ @quantumlytangled
```

Reply:
Alternatively you could leave a comment as to why you marked yourself as a codeowner of the whole `docker` folder